### PR TITLE
tex-fmt 0.4.5 (new formula)

### PR DESCRIPTION
**Pull request by ivaquero**
- [x] Have you followed the [guidelines for contributing](https://github.com/Homebrew/homebrew-core/blob/HEAD/CONTRIBUTING.md)?
- [x] Have you ensured that your commits follow the [commit style guide](https://docs.brew.sh/Formula-Cookbook#commit)?
- [x] Have you checked that there aren't other open [pull requests](https://github.com/Homebrew/homebrew-core/pulls) for the same formula update/change?
- [x] Have you built your formula locally with `HOMEBREW_NO_INSTALL_FROM_API=1 brew install --build-from-source <formula>`, where `<formula>` is the name of the formula you're submitting?
- [x] Is your test running fine `brew test <formula>`, where `<formula>` is the name of the formula you're submitting?
- [x] Does your build pass `brew audit --strict <formula>` (after doing `HOMEBREW_NO_INSTALL_FROM_API=1 brew install --build-from-source <formula>`)? If this is a new formula, does it pass `brew audit --new <formula>`?

-----

### CODE REVIEW

**Comment by ivaquero** [comment]
I need help, I don't know how to correct the error in `brew test`, it seems that `.write()` function didn't work.

File: Formula/t/tex-fmt.rb [hunk 1]
```diff
@@ -0,0 +1,39 @@
...
+      \\title{tex-fmt Homebrew Test}
+      \\begin{document}
+      \\maketitle
+      \\begin{itemize}
+      \\item Hello
+      \\item World
+      \\end{itemize}
+      \\end{document}
+    EOS
+    assert_match <<~EOS, shell_output("#{bin}/tex-fmt #{testpath}/test.tex")
```
Comment (chenrui333) [inline Formula/t/tex-fmt.rb:27]: this should work
```suggestion
    assert_equal <<~EOS, shell_output("#{bin}/tex-fmt --print #{testpath}/test.tex").chomp
```

**Review by chenrui333 — APPROVED** [review]

File: Formula/t/tex-fmt.rb (hunk 1 above)
Comment (ivaquero) [reply Formula/t/tex-fmt.rb]: Thank you so much!

**Review by nandahkrishna — APPROVED** [review]

**Comment by chenrui333** [comment]
@ivaquero, thanks for your contribution to Homebrew! :tada: 🥇

Without awesome contributors like you, it would be impossible to maintain Homebrew to the high level of quality users have come to expect. Thank you!!!!

**Pull request merged**